src/Compilers/CSharp/Portable/Syntax/InternalSyntax/CSharpSyntaxNode.cs

```diff
@@ -78,21 +78,8 @@ public override int RawContextualKind
             }
         }
 
-        public override bool IsStructuredTrivia
-        {
-            get
-            {
-                return this is StructuredTriviaSyntax;
-            }
-        }
```

Comment:
this code was very odd.  it was an override at a higher layer that then went and did a type check on 'this'.   Unfortunately, going back in source, i don't see any reasoning given for being written this way.  

Changed to just normal overrides where only the specialized subtypes themselves return true (instead of needing the typecheck here).